typings/modules/apollo-client/index.d.ts

```diff
@@ -496,13 +496,16 @@ export interface UpdateQueryOptions {
 }
 export class ObservableQuery extends Observable<ApolloQueryResult> {
     refetch: (variables?: any) => Promise<ApolloQueryResult>;
+    setVariables: (variables: any) => Promise<ApolloQueryResult>;
```

Comment:
I manually copied this over from `apollo-client` -- I'm not sure what the correct procedure is.